frontend/src/APIClients/CommonAPIClient.ts

```diff
@@ -0,0 +1,39 @@
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+import baseAPIClient from "./BaseAPIClient";
+
+const filterLogRecords = async (
+  building: string,
+  employeeId: string[],
+  attnTo: string[],
+  dateRange: string[],
+  tags: string[],
+  flagged: boolean,
+): Promise<any> => {
+  try {
+    const bearerToken = `Bearer ${getLocalStorageObjProperty(
+      AUTHENTICATED_USER_KEY,
+      "accessToken",
+    )}`;
+    const { data } = await baseAPIClient.get("/log_records", {
+      params: {
+        filters: {
+          building,
+          employeeId,
+          attnTo,
+          dateRange,
+          tags,
+          flagged,
+        },
+      },
+      headers: { Authorization: bearerToken },
+    });
+    return data;
+  } catch (error) {
+    return null;
```

Comment:
More of a later thought but we may want to distinguish between when there was an error with getting the logs vs. there's no logs depending on the design of the logs table

Reply:
will add a todo for future devs :DDDDD